examples/vision/learnable_resizer.py

```diff
@@ -198,7 +198,12 @@ def learnable_resizer(
 
 
 def get_model():
-    backbone = tf.keras.applications.DenseNet(weights=None, include_top=True, classes=2)
+    backbone = tf.keras.applications.DenseNet121(
+        weights=None,
+        include_top=True,
+        classes=2,
+        input_shape=((INP_SIZE[0], INP_SIZE[1], 3)),
```

Comment:
`input` of `backbone` is `output` of `learnable_resizer`, considering this the `input` of `backbone` must not be <br>
`((INP_SIZE[0], INP_SIZE[1], 3))`  instead it should be <br> `((TARGET_SIZE[0], TARGET_SIZE[1], 3))`

Reply:
Right. Corrected it.